bakerydemo/blog/models.py

```diff
@@ -41,9 +41,9 @@ class BlogPeopleRelationship(Orderable, models.Model):
 
 class BlogPageTag(TaggedItemBase):
     """
-    This model allows us to creates a many-to-many relationship between 
+    This model allows us to creates a many-to-many relationship between
```

Comment:
Good catch. Thanks! I'm finding it amazing adding these comments how dependent I am on spell-check 0_o